src/client/client.d

```diff
@@ -312,8 +312,16 @@ Socket createSocket(string socketFile, ushort port)
 	}
 	else
 	{
-		socket = new Socket(AddressFamily.UNIX, SocketType.STREAM);
-		socket.connect(new UnixAddress(socketFile));
+		version(Windows)
+		{
+			fatal("Cannot use UNIX domain sockets on Windows.");
+			assert(0);
```

Comment:
Can you replace this with a `return 1;` or an exception? Asserts should not be used to validate user input.